buildenv/docker/mkdocker.sh

```diff
@@ -372,6 +372,7 @@ fi
   echo "    perl-devel \\"
   echo "    perl-ExtUtils-MakeMaker \\" # required by git
   echo "    perl-GD \\"
+  echo "    perl-IPC-Cmd \\" # required for openssl v3 compiles
```

Comment:
If I remember correctly, it was required to compile openssl 1.1.1s, so the "v3" qualification should be removed.

Reply:
We're already building 1.1.1s and it's working with the 'eclipseopenj9/jenkins-agent-x86-centos6:16' container, so it must not be needed.

Reply:
What versions do we build in a docker container? I couldn't even run `configure` for jdk17 successfully in `eclipseopenj9/jenkins-agent-x86-centos6:16`:
```
checking for version string... 17.0.6-internal+0-adhoc..jdk17
configure: Found potential Boot JDK using configure arguments
configure: Potential Boot JDK found at /home/jenkins/bootjdks/jdk16 is not a working JDK; ignoring
configure: Output from java -version was: /home/jenkins/bootjdks/jdk16/bin/java: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /home/jenkins/bootjdks/jdk16/bin/../lib/libjli.so)
configure: error: The path given by --with-boot-jdk does not contain a valid Boot JDK
```

Reply:
> What versions do we build in a docker container?

jdk8 and 11, as per the defaults.yml
```
    docker_image:
      8: 'eclipseopenj9/jenkins-agent-x86-centos6:24'
      11: 'eclipseopenj9/jenkins-agent-x86-centos6:24'
```